packages/yourloops/components/dialogs/remote-monitoring-invite.tsx

```diff
@@ -87,12 +92,26 @@ function RemoteMonitoringPatientInviteDialog(props: RemoteMonitoringPatientInvit
   };
   const [saveButtonDisabled, setSaveButtonDisabled] = useState(true);
 
-  const onSave = () => {
+  const onSave = async () => {
     console.log(prescriptionInfo.teamId);
     console.log(prescriptionInfo.memberId);
     console.log(prescriptionInfo.file);
     console.log(prescriptionInfo.numberOfMonth);
     console.log(physician);
+    const monitoringEnd = new Date();
+    monitoringEnd.setMonth(monitoringEnd.getMonth() + 3);
```

Comment:
Why not directly use prescriptionInfo.numberOfMonth ? 

maybe using moment().add(numberOfMonth, 'm') would be simpler (and avoid corner cases with end of month/enf of year etc.)